shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -671,6 +656,22 @@ public void onMessage(Object message, BasicMessageChannel.Reply<Object> reply) {
             case "announce":
                 mOwner.announceForAccessibility((String) data.get("message"));
                 break;
+            case "longPress": {
+                if (mA11yFocusedObject == null) {
+                    return;
+                }
+                AccessibilityEvent e = obtainAccessibilityEvent(mA11yFocusedObject.id, AccessibilityEvent.TYPE_VIEW_LONG_CLICKED);
```

Comment:
Can we obtain the id from the event? The focus might have changed between clicking the button and dispatching this event.

Reply:
Is there a way to obtain a semantics node's id from the framework side?

Reply:
There is, but it's not easy. We used to do this for the "scroll happened" event, which was removed in https://github.com/flutter/flutter/pull/14536. Maybe the old code their is inspiration how to get the id?

Reply:
Actually, we could use the render object, grab the center and then pass the offset back to the AccessibilityBridge.  in theory hit testing the semantics objects there would give us the correct id, thoughts? @goderbauer

Reply:
We could have overlapping semantics objects. In that case hit testing could potentially yield the wrong object.

Reply:
Can mA11yFocusedObject be null?

Reply:
(You should assume that it is possible that the framework side is hostile and will call this API willy nilly.)

Reply:
Heh, so you are. I was only looking at the very line here, my bad!